src/chrome-api/index.js

```diff
@@ -18,6 +18,15 @@ export function create() {
 var ChromeManager = {
 
     /**
+     * Install plugin
+     * @param {Object} plugin
+     * @param {Function} plugin.install
+     */
+    install: function (plugin) {
+        plugin.install(this);
```

Comment:
No, I think it will break `sinon-chrome` concept.
Every new plugin will break backward compatibility.

All chrome methods are stubs, by default. And you have ability to change this behavior by installing plugin.